docs/administer/custom_policies.md

```diff
@@ -0,0 +1,131 @@
+# Custom Pod scheduling policies
+
+Percona Everest allows you to define custom Pod scheduling policies to control how database pods are placed on Kubernetes nodes.
+
+## Create Pod scheduling policy for your DB cluster
+
+Here are the steps to configure pod scheduling rules for the policies for your database clusters:
+{.power-number}
+
+1. From the Percona Everest home page, navigate to the <i class="uil uil-cog"></i> **Settings > Pod scheduling policies** page. Here, you can view both default and custom policies.
+
+
+    ![!image](../images/pod_scheduling_policies.png)
+
+
+2. Click **Create policy**.
+
+3. In the pop-up that appears, enter a **Policy name** and select the **database technology** from the dropdown. Click **Create**.
+
+
+4. Click **Add rule**.
+
+5. A pop-up will appear where you need to enter the following details for the **Rule type** and **Rule details** section:
+
+    ![!image](../images/affinity_add_rules.png)
+
+    Refer to the following table for the detailed attribute descriptions.
+
+    ??? info "📋 Custom Pod scheduling rule attributes"
+        The table below describes the key attributes used to define pod scheduling rules in Percona Everest:
+
+
+        | **Attribute**     | **Description** | **Comments** |
+        |------------------|-----------------|-----------|
+        | **Components**    | The database cluster components the rule applies to:<br>- **DB Node**<br>- **Proxy / Router / PgBouncer**<br>- **Config Server** | - DB Nodes and Proxies are applicable for **MySQL** and **PostgreSQL**.<br>- Config Servers apply to **MongoDB** sharded clusters. |
+        | **Priority**      | Defines the distinct level of rule enforcement:<br>- **Preferred**: Percona Everest will *try* to honor the rule but will schedule the pod even if it's not met.<br>- **Required**: The rule *must* be satisfied for the pod to be scheduled. | Use **Preferred** for flexible placement and **Required** for hard constraints. |
```

Comment:
`Percona Everest will *try* to honor the rule`- Everest does not deal with these policies or rules. this a Kubernetes that uses rules during choosing the node for running pod

Reply:
Fixed